fsharp-backend/src/LibBackend/EventQueue.fs

```diff
@@ -155,18 +155,32 @@ module WorkerStates =
 
 
 
-// None in case we want to log on a timer or something, not
-// just on enqueue/dequeue
+/// <summary>
+/// Logs the current status of what's enqueued for the the given queue,
+/// both at the "worker level" (ready to be picked up by workers),
+/// and at the "canvas level" (ready to be enqueued _for_ a worker).
+/// </summary>
+///
+/// <remarks>
+/// `host` is optional because we have it when we dequeue, but not enqueue.
+/// </remarks>
+///
+/// WHATISTHIS why do we have both 'name' and 'host'?
+/// is it anticipation of a queue existing in one canvas, but being used in another?
 let logQueueSize
   (queue_action : queue_action)
+
+  // better name ideas: anchorCanvas, or managedBy, or ownedBy.
+  // esp. useful if we switch to a "CanvasName of string" type here, rather than just string.
+  // then this could be `(managedBy: CanvasName option)`
   (host : Option<string>)
+
   (canvasID : CanvasID)
   (space : string)
-  (name : string)
+  (queueName : string)
```

Comment:
This name isn't used anywhere else. I don't hate it but we should probably keep it as "name". An alternative could be to switch this to a PT.Handler.Spec.Cron?